### PR TITLE
6x dependencies

### PR DESCRIPTION
**Pull request by cbeer**

### CODE REVIEW

**Comment by jcoyne** [comment]
You may need to update the ruby versions in travis to allow this to build.

File: Gemfile [hunk 1]
```diff
@@ -10,3 +10,6 @@ group :development, :test do
 end
 
 gem 'jruby-openssl', :platform=> :jruby
+gem 'rubydora', path: '~/Projects/hydra/rubydora'
```
Comment (cbeer) [inline Gemfile]: But the build passes there!

**Comment by mjgiarlo** [comment]
Wait, I'm confused; I was promised a copy of @cbeer's laptop with this PR.

**Comment by cbeer** [comment]
Given my luck the last couple weeks, I'm not sure you want it.

**Pull request merged**